wiki/markdown/rules.md

```diff
@@ -142,7 +142,7 @@ Use marker "%IdentifyRules_objid%" to inject the ids generated by
 **_IdentifyRules_**. Examples:
 
     {"_id": "%IdentifyRules_objid%"},
-    { "$and": [{"aggregatedObject.testCaseExecutions.testCaseStartedEventId": "%IdentifyRules%"}]}
+    { "$and": [{"testCaseExecutions.testCaseStartedEventId": "%IdentifyRules%"}]}
```

Comment:
Somewhere in this page we need to add a new section mention which keys are reserved by Eiffel Intelligence. 

"When the user defines their own rules, they also define what keys to use in structuring the aggregated object. Eiffel Intelligence stores all aggregated objects in a mongo DB database. When it does so, some keys are added to the root level of the aggregated object, and if the user has defined these in their rules they will be overwritten with values by Eiffel Intelligence. These reserved keys are "**_id**" and "**time**"."

Don't know if I forgot any other reserved keys, but those are the ones I see right now...

Reply:
Also, I want each section on this page to have it's own header, marked with \## instead of just bold formatting. It makes it easier to find and read the page, and it also enables ut to use permalinks to specific sections of this page.

Reply:
If we mentioned that these keys are reserved what happens if they use them anyway? I don't think you can change a _id though.

Reply:
If user use these keys they will be overwritten by Eiffel Intelligence.

Reply:
Added in the doc that "_id" and "time" are reserved at the root level. Regarding header for each section..... bold formatting representing the hierarchical order ...?

Reply:
sections representing keys are at ### level now...?

Reply:
Maybe just have one header for reserved keys, and list them under that one.